extensions/commands/art/cmd_build_info.py

```diff
@@ -389,7 +389,8 @@ def build_info_create(conan_api: ConanAPI, parser, subparser, *args):
     data = load_json(args.json)
 
     # remove the 'conanfile' node
-    data["graph"]["nodes"].pop("0")
+    if data["graph"]["nodes"]["0"]["ref"] == "conanfile":
+        data["graph"]["nodes"].pop("0")
```

Comment:
```suggestion
    if not data["graph"]["nodes"]["0"]["prev"]:
        data["graph"]["nodes"].pop("0")
```
I think this should work well in all cases